packages/core/upload/server/utils/file.js

```diff
@@ -5,7 +5,14 @@
  */
 const { Writable } = require('stream');
 
+const kbytesToBytes = (kbytes) => kbytes * 1000;
 const bytesToKbytes = (bytes) => Math.round((bytes / 1000) * 100) / 100;
+const bytesToHumanReadable = (bytes) => {
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB'];
+  if (bytes === 0) return '0 Bytes';
+  const i = parseInt(Math.floor(Math.log(bytes) / Math.log(1000)), 10);
+  return `${Math.round(bytes / 1000 ** i, 2)} ${sizes[i]}`;
```

Comment:
the famous question should you divide by 1000 or 1024 ? :D

Reply:
I was wondering the same just now. I went for 1000 as the other functions were using that base. 
What do you think should we use?

Reply:
As long as you keep this consistent I'm fine with 1000. I think mac uses 1000 and windows 1024 or the other way around so 🤷